src/DittoMachine.sol

```diff
@@ -314,14 +318,26 @@ contract DittoMachine is ERC721, ERC721TokenReceiver, ERC1155TokenReceiver, Clon
                 _amount
             );
             // buying out the previous clone owner
+
             address curOwner = ownerOf[cloneId];
-            uint256 subsidyDiv2 = subsidy >> 1;
+
+            // subtract subsidy refund from subsidy pool
+            cloneIdToSubsidy[cloneId] -= (feeRefund);
+
+            // uint256 subsidyDiv2 = (subsidy >> 1);
+            {
+                _setBlockRefund(cloneId, subsidy);
+            }
             // half of fee goes into subsidy pool, half to previous clone owner
-            cloneIdToSubsidy[cloneId] += subsidyDiv2;
+            cloneIdToSubsidy[cloneId] += feeRefund == 0 ? (subsidy >> 1) : subsidy;
```

Comment:
shouldn't it still just be subsidy >> 1?

Reply:
if another bid was in the block then the subsidy that was refunded will need to be replaced